lib/internal/Magento/Framework/GraphQl/Schema/SchemaGenerator.php

```diff
@@ -55,16 +54,15 @@ public function generate() : Schema
     {
         $schema = $this->schemaFactory->create(
             [
-                'query' => $this->outputMapper->getOutputType('Query'),
-                'mutation' => $this->outputMapper->getOutputType('Mutation'),
+                'query' => $this->typeRegistry->get('Query'),
+                'mutation' => $this->typeRegistry->get('Mutation'),
                 'typeLoader' => function ($name) {
-                    return $this->outputMapper->getOutputType($name);
+                    return $this->typeRegistry->get($name);
                 },
                 'types' => function () {
-                    //all types should be generated only on introspection
                     $typesImplementors = [];
-                    foreach ($this->config->getDeclaredTypeNames() as $name) {
-                        $typesImplementors [] = $this->outputMapper->getOutputType($name);
+                    foreach ($this->config->getDeclaredTypeNames() as $type) {
```

Comment:
Please create a ticket not to load all types, but just the ones which do not have references from Query/Mutation types